c_src/membrane_h264_ffmpeg_plugin/encoder.c

```diff
@@ -123,6 +123,11 @@ UNIFEX_TERM create(UnifexEnv *env, int width, int height, char *pix_fmt,
 
   set_x264_defaults(&params, preset);
 
+  for(unsigned int i=0; i<ffmpeg_options_length; i++)
+  {
+    av_dict_set(&params, ffmpeg_options[i].key, ffmpeg_options[i].value, 0);
+  }
```

Comment:
make sure that these options override the other options

Reply:
It indeed does ;)